docs/pool-manager.md

```diff
@@ -0,0 +1,74 @@
+# What is the Pool Manager and what does it do?
+
+👉 A component that provisions and manages vLLM pods within the Dual-Pod (DP) Controller architecture. Can be a Kubernetes native service that maintains a pool of idle pods running just the launcher, pre-labeled for IGW’s InferencePool, and transitions them through some different lifecycle states (idle → active → sleeping → deleted) to ensure fast activation of server-running vLLM pods while minimizing cold-start latency.
+
+Flow may look like this for create:
+
+- EPP creates a ISR pod,
+- DP Controller watches it and asks Pool Manager for a inference server pod,
```

Comment:
Both the user/client-created Pod and the one that actually runs the inference server can be described as "inference sever pods". That is why I use two terms to distinguish them: server-requesting Pod and server-running Pod.